src/components/icons/archive.tsx

```diff
@@ -0,0 +1,18 @@
+export default function Archive() {
```

Comment:
Could you kindly explain why **arrow function** notation is not considered for the components located within the **icons** folder (**archive, delete, user-plus**)? Specifically, I'm wondering if this is related to the "**currentColor**" props and the need to access it within the component's lifecycle using '**this**'. 

I would greatly appreciate your insights on this matter. Thank you in advance for your input.